### ISSUE TITLE
COCO ReXNet Model

### ISSUE THREAD

**Issue by debparth** [clovaai/rexnet#22]
Hi @dyhan0920 and other Contributors. 
Thank you for making public such a good repo. 

It's totally amazing that ReXNet models are better than Efficient Net models. Looks like we got another SOTA model!!! 

I saw the profiling in ImageNet & COCO dataset. I want to try evaluating those models but was not able to for COCO. So, can you provide the weights for COCO ReXNet version models?

**Comment by dyhan0920** [comment]
Sorry for the late reply @debparth. Do you need both of SSDLite and Faster RCNN models? I will clean the codes quickly and update them soon. Thank you for your interest!

**Comment by debparth** [comment]
@dyhan0920 It'd be great if you can provide both models. Thanks for making such a good repo public.

**Comment by debparth** [comment]
@dyhan0920 I can't wait to test the SOTA models. so, when will you update the models?

**Comment by dyhan0920** [comment]
@debparth Cleaning up the detection code is near the top of my priority but I couldn't manage to do it on time. Very sorry for the late update. Before update the code I trained, I rather provide the baseline training code.

SSDLite: I have trained ReXNets inside SSD-lite with two training codes and confirmed both codes yield almost identical results. [This code]((https://github.com/Andrew-Qibin/ssdlite-pytorch-mobilenext) seems to be quickly applicable to any backbone architectures than [the other one](https://github.com/amdegroot/ssd.pytorch). 

Faster/Mask-RCNN: I have trained ReXNets in [mmdetection](https://github.com/open-mmlab/mmdetection) and Torchvision's Faster/Mask-RCNN code (see this [code](https://pytorch.org/vision/stable/_modules/torchvision/models/detection/faster_rcnn.html) which is very similar to the FAIR's detection code. I mainly used the later code at last but both training codes produced very similar AP scores.

On each baseline, one thing you need to do is to settle where to attach the detection heads, which followed the rule-of-thumb in the competitive method. Please refer to the details in our paper.
